integration/integration_test.go

```diff
@@ -225,7 +225,7 @@ func checkLayers(image1, image2 string, offset int) error {
 	}
 	actualOffset := int(math.Abs(float64(lenImage1 - lenImage2)))
 	if actualOffset != offset {
-		return fmt.Errorf("incorrect offset between layers of %s and %s: expected %d but got %d", image1, image2, offset, actualOffset)
+		return fmt.Errorf("Difference in number of layers in each image is %d but should be %d. %s has %d layers and %s has %d layers", actualOffset, offset, image1, lenImage1, image2, lenImage2)
```

Comment:
How tough would it be to dump the full image digest as part of this? That way we could debug later.

Reply:
IT IS DONE